### PR TITLE
--downloader.verfiy flag to verify once on startup

### PR DESCRIPTION
**Pull request by AskAlexSharov**
- it will not report founded problems but just re-download broken pieces
- maybe rename it to --downloader.repair? 

- Erigon will also verify once after downloading all files done (even without flag)

### CODE REVIEW

File: cmd/downloader/downloader/downloader.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ledgerwatch/erigon/common"
 	"github.com/ledgerwatch/log/v3"
 	mdbx2 "github.com/torquem-ch/mdbx-go/mdbx"
+	"go.uber.org/atomic"
```
Comment (AlexeyAkhunov) [inline cmd/downloader/downloader/downloader.go:22]: Are you sure you don't want `sync/atomic` instead?

File: cmd/downloader/downloader/downloader.go [hunk 2]
```diff
@@ -325,12 +371,17 @@ func MainLoop(ctx context.Context, d *Downloader, silent bool) {
...
 				t.DownloadAll()
 				go func(t *torrent.Torrent) {
+					defer sem.Release(1)
 					//r := t.NewReader()
 					//r.SetReadahead(t.Length())
 					//_, _ = io.Copy(io.Discard, r) // enable streaming - it will prioritize sequential download
 
 					<-t.Complete.On()
-					sem.Release(1)
+					go func(t *torrent.Torrent) {
```
Comment (AlexeyAkhunov) [inline cmd/downloader/downloader/downloader.go:380]: Why do you need nested `go func` here? Perhaps you mean to do one of them without `go` to use `defer sem.Release()` ?

Comment (AskAlexSharov) [reply cmd/downloader/downloader/downloader.go:380]: You right. I don’t need. It’s trash after experiment.

File: cmd/downloader/downloader/downloader.go (hunk 1 above)
Comment (AskAlexSharov) [reply cmd/downloader/downloader/downloader.go:22]: Yes. It’s package with typed wrappers for atomic.

**Pull request merged**